protocol/v2/ssv/runner/committee.go

```diff
@@ -45,7 +46,9 @@ type CommitteeRunner struct {
 	postStarted   time.Time
 }
 
-func NewCommitteeRunner(beaconNetwork types.BeaconNetwork,
+func NewCommitteeRunner(
+	networkConfig networkconfig.NetworkConfig,
+	beaconNetwork types.BeaconNetwork,
```

Comment:
I guess `BeaconNetwork` isn't needed anymore now that `NetworkConfig` is passed, since `NetworkConfig` has it. 
either pass in exactly what `CommitteeRunner` uses out of `NetworkConfig` or just pass in `NetworkConfig` as whole and use it.